src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -486,6 +503,52 @@ private GoodDateChooser getJCalendarFrom() {
 		return jCalendarInventory;
 	}
 
+	private JButton getSelectedButton() {
+		if (selectButton == null) {
+			selectButton = new JButton(MessageBundle.getMessage("angal.common.select.btn"));
+			selectButton.setMnemonic(MessageBundle.getMnemonic("angal.common.select.btn.key"));
+			selectButton.addActionListener(actionEvent -> {
+				specificRadio.setSelected(false);
+		        mainPanel = new JPanel();
```

Comment:
the formatting is off starting here

Reply:
I don't have the same behavior localy
![bug](https://github.com/user-attachments/assets/5745141c-8916-4ac6-bfbd-d488f913c075)

Reply:
To quote @mwithi it doesn't matter how it looks locally but how it looks on github:
![image](https://github.com/user-attachments/assets/6d8d2c5e-1c17-4082-95b0-0f82cc76992a)

You can see this by viewing the file in the changed files list on the right side:
![image](https://github.com/user-attachments/assets/191d6fc6-2aec-4587-a6c7-d6bddfa9ecb0)

FWIW, the link is: https://github.com/informatici/openhospital-gui/blob/9fb1f94bbc1c6e47025192d9950fa0bbbd9b9cb2/src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java